libzfs.pyx

```diff
@@ -2705,22 +2745,23 @@ cdef class ZFSDataset(ZFSObject):
         if err != 0:
             raise self.root.get_error()
 
-    def get_send_progress(self, fd):
-        cdef zfs.zfs_cmd_t cmd
-        cdef int ret
+    IF HAVE_ZFS_IOCTL_HEADER:
```

Comment:
I think I would rather have this method always defined but raise a NotImplementedError if not available.